e3/vcs/svn.py

```diff
@@ -89,6 +137,8 @@ def get_info(self, item):
         """
         info = self.svn_cmd(['info'], output=PIPE).out
         m = re.search(r'^{item}: *(.*)\n'.format(item=item), info, flags=re.M)
+        if m is None:
```

Comment:
Other than the logging, I don't see any specific handling of that case, meaning we'll get to line 142, which should result in a crash because 'm' is None. Can we add a test in the testsuite to check for that?